offload/plugins-nextgen/common/src/RPC.cpp

```diff
@@ -93,6 +94,22 @@ Error RPCServerTy::runServer(plugin::GenericDeviceTy &Device) {
     });
     break;
   }
+  case OFFLOAD_HOST_CALL: {
+    uint64_t Sizes[64] = {0};
```

Comment:
What's the `64` here? Wave size?

Reply:
I was wondering if we at least should have a comment there what it is, since it may not be obvious for everybody.

Reply:
Done